src/UI/Settings/Notifications/Edit/NotificationEditViewTemplate.hbs

```diff
@@ -77,24 +77,26 @@
                         </label>
 
                         <span class="help-inline-checkbox">
-                            <i class="icon-sonarr-form-info" title="Do you want to get notifications when episodes are upgraded to a better quality?"/>
+                            <i class="icon-sonarr-form-info" title="Be notified when episodes are upgraded to a better quality"/>
                         </span>
                     </div>
                 </div>
             </div>
 
             <div class="form-group">
-                <label class="col-sm-3 control-label">Tags</label>
+                <label class="col-sm-3 control-label">Filter Series Tags</label>
```

Comment:
I think for consistency this should remain Tags, we can use to the info tooltip to explain it.